CHANGELOG.md

```diff
@@ -1,7 +1,11 @@
 # Changes
 This file documents all changes to Argus-frontend. This file is primarily meant to be read by developers.
 
-## [Unreleased]
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/), and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
+This project uses [*towncrier*](https://towncrier.readthedocs.io/) and the changes for the upcoming release can be found in <https://github.com/Uninett/Argus-frontend/tree/master/changelog.d/>.
```

Comment:
IMHO, a relative link to the current repo is better (also, we don't have to change it later, when the github URL changes):

```suggestion
This project uses [*towncrier*](https://towncrier.readthedocs.io/) and the changes for the upcoming release can be found in the [changelog.d/](./changelog.d/) directory.
```